[2025-02-23T04:28:29.634+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-23T04:28:29.674+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag.first_task scheduled__2021-06-30T00:00:00+00:00 [queued]>
[2025-02-23T04:28:29.698+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag.first_task scheduled__2021-06-30T00:00:00+00:00 [queued]>
[2025-02-23T04:28:29.699+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 6
[2025-02-23T04:28:29.739+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): first_task> on 2021-06-30 00:00:00+00:00
[2025-02-23T04:28:29.755+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1045) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-23T04:28:29.758+0000] {standard_task_runner.py:72} INFO - Started process 1051 to run task
[2025-02-23T04:28:29.755+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag', 'first_task', 'scheduled__2021-06-30T00:00:00+00:00', '--job-id', '151', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmp0jbg91o5']
[2025-02-23T04:28:29.760+0000] {standard_task_runner.py:105} INFO - Job 151: Subtask first_task
[2025-02-23T04:28:29.859+0000] {task_command.py:467} INFO - Running <TaskInstance: our_first_dag.first_task scheduled__2021-06-30T00:00:00+00:00 [running]> on host f66dd6409c88
[2025-02-23T04:28:30.052+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='vn5705j' AIRFLOW_CTX_DAG_ID='our_first_dag' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2021-06-30T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-06-30T00:00:00+00:00'
[2025-02-23T04:28:30.054+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-23T04:28:30.101+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-02-23T04:28:30.103+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', "echo 'Hello World!' This is the first Task"]
[2025-02-23T04:28:30.120+0000] {subprocess.py:99} INFO - Output:
[2025-02-23T04:28:30.124+0000] {subprocess.py:106} INFO - Hello World! This is the first Task
[2025-02-23T04:28:30.125+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-02-23T04:28:30.176+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-23T04:28:30.182+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=our_first_dag, task_id=first_task, run_id=scheduled__2021-06-30T00:00:00+00:00, execution_date=20210630T000000, start_date=20250223T042829, end_date=20250223T042830
[2025-02-23T04:28:30.299+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-23T04:28:30.329+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-23T04:28:30.332+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
